go.mod

```diff
@@ -191,7 +190,7 @@ require (
 replace (
 	// Ref: https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.7
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.9-0.20230130122755-5e7dcbcce1e9
```

Comment:
is this issue related to the iavl update? 0.19.4 -> 0.19.5.RC1

Reply:
yes, fixed here: https://github.com/cosmos/iavl/pull/640